PlaceRouteHierFlow/placer/Placer.h

```diff
@@ -17,7 +17,7 @@ using std::endl;
 //#define MAX_TIMEOUT 4300000 //4.3 seconds = 4300000 us
 #define T_INT 1e6
 #define T_MIN 1e-6
-#define ALPHA 0.95
```

Comment:
These constants (hyperparameters) should be brought under Python control so then can be changed without having to recompile the code. Do you want me to do that for you in this PR?

Reply:
@stevenmburns Sure. Please go ahead.